content/guides/models/registry/registry_cards.md

```diff
@@ -0,0 +1,78 @@
+---
+menu:
+  default:
+    identifier: registry_cards
+    parent: registry
+title: Document collections
+weight: 9
+---
+
+Add human-friendly text to your collections to help users understand the purpose of the collection and the artifacts it contains. 
+
+
+Depending on the collection, you might want to include information about the training data, model architecture, task, license, references, and deployment. The proceeding lists some topics worth documenting in a collection:
+
+
+
+In general, a collection description should include:
```

Comment:
```suggestion
We recommend that you include at minimum these details:
```

Reply:
Isn't "we" a sin in tech docs...?  Maybe "W&B recommends..." instead?

Reply:
Sure, I edited my edit. (I think you can also edit a reviewer's edit)